native/components/Navbar.jsx

```diff
@@ -0,0 +1,77 @@
+//utilities
+import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+//screens
+import { Applicants } from '../screens/Applicants/Applicants';
```

Comment:
a lot of these screens are imported but not used in this component, let's remove them to keep this file clean from dead/stale code

Reply:
I completely agree. Thanks for bringing to my attention! Will keep this in mind in the future